atom/common/crash_reporter/crash_reporter_mac.mm

```diff
@@ -67,7 +67,7 @@
   simple_string_dictionary_.reset(new crashpad::SimpleStringDictionary());
   crashpad_info->set_simple_annotations(simple_string_dictionary_.get());
 
-  SetCrashKeyValue("prod", CHROMIUM_SHORT_NAME);
+  SetCrashKeyValue("prod", ATOM_PRODUCT_NAME);
```

Comment:
happy to update it but couple notes:
0.12 had ATOM_PRODUCT_NAME, and I changed it to be consistent with what linux uses:
atom/common/crash_reporter/crash_reporter_linux.cc:  crash_keys_.SetKeyValue("prod", ATOM_PRODUCT_NAME);

So if I change to the CHROMIUM constant then I should update linux too.